opendut-carl/Cargo.toml

```diff
@@ -15,34 +15,42 @@ opendut-util = { workspace = true }
 opendut-vpn = { workspace = true }
 
 anyhow = { workspace = true }
-tempfile = { workspace = true }
 axum = { workspace = true }
 axum-server = { workspace = true, features = ["tls-rustls"] }
 axum-server-dual-protocol = { workspace = true }
+base64 = { workspace = true }
+chrono = { workspace = true }
 config = { workspace = true }
 flate2 = { workspace = true }
 futures = { workspace = true }
 googletest = { workspace = true }
 http = { workspace = true }
 indoc = { workspace = true }
 itertools = { workspace = true }
-pem = { workspace = true, features = ["serde"]}
+jsonwebtoken = { workspace = true}
+openidconnect = { workspace = true }
 opentelemetry = { workspace = true }
 opentelemetry_sdk = { workspace = true }
+pem = { workspace = true, features = ["serde"]}
+reqwest = { workspace = true, features = ["json"] }
 serde = { workspace = true, features = ["derive"] }
+serde_json = { workspace = true}
 shadow-rs = { workspace = true, default-features = true }
 tar = { workspace = true }
+tempfile = { workspace = true }
 thiserror = { workspace = true }
 tokio = { workspace = true, features = ["full"] }
 tokio-stream = { workspace = true, features = ["full"] }
 tonic = { workspace = true, features = ["default"] }
 tonic-web = { workspace = true }
+tonic-async-interceptor = { workspace = true }
 tower = { workspace = true, features = ["make", "steer"] }
 tower-http = { workspace = true }
 tracing = { workspace = true }
 tracing-opentelemetry = { workspace = true }
 url = { workspace = true, features = ["serde"] }
 uuid = { workspace = true }
+log = "0.4.22"
```

Comment:
This dependency should be ordered alphabetically and reference the workspace.

Reply:
Removed import because it is not needed anymore